enclave_wrangler/objects_api.py

```diff
@@ -288,21 +288,23 @@ def all_new_objects_enclave_to_db(since: Union[datetime, str]) -> Dict[str, List
     return objects
 
 
-def csets_and_members_enclave_to_db(
-    con: Connection, schema: str, since: Union[datetime, str], filter_0_member_sets=True
-):
-    """Fetch new csets and members, if needed, and then update database with them."""
+def csets_and_members_enclave_to_db(con: Connection, schema: str, since: Union[datetime, str]) -> Union[None, str]:
+    """Fetch new csets and members, if needed, and then update database with them, returns None is no updates needed"""
     print('Fetching new data from the N3C data enclave...')
     t0 = datetime.now()
     csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(since)
+    if not csets_and_members:
+        return None
```

Comment:
I actually realized that `return None` is the same as `return`. They both return `None`. So I shortened this to just `return`.